tests/queries/0_stateless/03252_execute_as.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Tags: no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+cp ${CLICKHOUSE_CONFIG_DIR}/users.xml "$CURDIR"/users.xml
+sed -i 's/<!-- <access_management>1<\/access_management> -->/<access_management>1<\/access_management>/g' "$CURDIR"/users.xml
+
+server_opts=(
+    "--config-file=$CURDIR/$(basename "${BASH_SOURCE[0]}" .sh).config.xml"
+    "--"
+    # to avoid multiple listen sockets (complexity for port discovering)
+    "--listen_host=127.1"
+    # we will discover the real port later.
+    "--tcp_port=0"
+    "--shutdown_wait_unfinished=0"
```

Comment:
It doesn't make sense to start another server in this test.
`access_management` is [already set to 1](https://github.com/ClickHouse/ClickHouse/blob/master/tests/config/users.d/access_management.xml) in our CI.

Reply:
I rewrote the test and simplified it.